src/rdkafka_conf.c

```diff
@@ -739,6 +739,21 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           _RK(ssl.keystore_location),
           "Path to client's keystore (PKCS#12) used for authentication."
         },
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+        { _RK_GLOBAL, "ssl.engine.location", _RK_C_STR,
+          _RK(ssl.engine_location),
+          "Path to openssl engine library. OpenSSL >= 1.1.0 required."
+        },
+        { _RK_GLOBAL, "ssl.engine.id", _RK_C_STR,
+          _RK(ssl.engine_id),
+          "Openssl engine id. If parameter not provided along with ssl.engine.location, "
```

Comment:
Remove "If param.." sentence and simply set "dynamic" as the default value here. (see other str defaults)

Reply:
OpenSSL, not Openssl.

Also explain what the engine id is used for.